### PR TITLE
Apply costs to send projections

### PR DESCRIPTION
**Pull request by Eleonore9**

### CODE REVIEW

File: test/witan/send/send_test.clj [hunk 1]
```diff
@@ -556,3 +556,61 @@
...
+          ((juxt :total-population
+                 :current-year-in-loop) (append-to-total-population-1-0-0
+                                         {:total-population total-population
+                                          :current-population current-population
+                                          :current-year-in-loop current-year-in-loop}))
+          popn-projections (:send-projection (group-send-projection new-total-population))
+          send-costs (get-individual-input :cost-profile)
+          send-costs (:send-costs (apply-costs {:send-projection popn-projections :cost-profile send-costs}))]
+      (doall
+       (for [{:keys [need placement low-ci high-ci average-population cost] :as row} (ds/row-maps send-costs)]
```
Comment (Eleonore9) [inline test/witan/send/send_test.clj]: Good idea! :sunglasses:

File: test/witan/send/send_test.clj [hunk 2]
```diff
@@ -556,3 +556,61 @@
...
+             (do
+               (is (= :Non-SEND placement))
+               (is (= 0.0 cost)))
+
+             (= :UO need)
+             (do
+               (is (= 0.0 cost)))
+             :else
+             (do
+               (is (> cost 0.0) (pr-str row))))))))))
```
Comment (Eleonore9) [inline test/witan/send/send_test.clj]: Is there a difference between `str` and `pr-str`?

Comment (acron0) [inline test/witan/send/send_test.clj]: Yeah, `str` calls `toString` on the type whereas `pr-str` is the EDN representation

**Comment by Eleonore9** [comment]
:shipit:

**Pull request merged**